client/modules/wj-css/src/components/_scrollbar.scss

```diff
@@ -0,0 +1,35 @@
+// Avoid setting scrollbars for something that's probably a phone
+@include media(">narrow") {
+  * {
+    scrollbar-width: thin;
```

Comment:
Comment for the file: why can't we just let browsers do scrollbars? Requesting changes

Reply:
Which do you prefer?

![image](https://user-images.githubusercontent.com/34875062/117294980-86e8d080-ae30-11eb-9f23-8a97be633c62.png)

![image](https://user-images.githubusercontent.com/34875062/117295002-8c461b00-ae30-11eb-9532-a1d4aeaf9e05.png)

The default browser scrollbars are inconsistent native implementations, and they're usually _gigantic_. Additionally, _I am_ leaving it up to the browser when I can - the `scrollbar-width: thin` property is the "official" way of doing this.

Reply:
https://webaim.org/resources/contrastchecker/?fcolor=44464C&bcolor=282B34

The default implementations are consistent for a single user across whatever websites they visit. It doesn't matter if they vary between browsers.

It doesn't really matter to me if an individual site wants a special scrollbar or if Sheaf wants a special scrollbar, but this is the base theme, and it defines everything else. I'd like to defer to the defaults that the user expects as much as possible.

Reply:
Yes, you pointed out the color - but that wasn't really the point? We can adjust the color easily?

I'm also going to fight you on that "consistency" point - who browses a site with a single device nowadays? On Macs, Safari will often just plain _disable_ scrollbars. I think this brings them back - I'd have to test that.

I also don't think the user has much of an idea of what a "default scrollbar" is - nearly every application they will be using has a different one. Even different Windows settings apps have different scrollbars.

Reply:
Hmmm. Alright, fair enough. I'll concede to you on that last.